docs/excel/custom-functions-web-reqs.md

```diff
@@ -119,36 +119,55 @@ ws.onerror(error){
 }
 ```
 
-## Streaming functions
+## Streaming and cancelable functions
 
-Streaming custom functions enable you to output data to cells repeatedly over time, without requiring a user to explicitly request data refresh. The following code sample is a custom function that adds a number to the result every second. Note the following about this code:
+Streaming custom functions enable you to output data to cells repeatedly over time, without requiring a user to explicitly request data refresh.
 
-- Excel displays each new value automatically using the setResult callback.
-- The second input parameter, handler, is not displayed to end users in Excel when they select the function from the autocomplete menu.
-- The onCanceled callback defines the function that executes when the function is canceled. You must implement a cancellation handler like this for any streaming function. For more information, see [Canceling a function](#canceling-a-function).
+Cancelable custom functions enable you to cancel the execution of a streaming custom function to reduce its bandwidth consumption, working memory, and CPU load. Excel automatically cancels the execution of a function in the following situations:
+
+- When the user edits or deletes a cell that references the function.
+- When one of the arguments (inputs) for the function changes. In this case, a new function call is triggered following the cancellation.
+- When the user triggers recalculation manually. In this case, a new function call is triggered following the cancellation.
```

Comment:
I had some comments related to this in PR #834. Make sure to transfer any changes here.